internal/ghworkflow/defaults.go

```diff
@@ -20,8 +20,9 @@ import "strings"
 // Constants
 
 const (
-	defaultPostgresVersion = "12"
-	defaultRunnerOS        = "ubuntu-latest"
+	defaultPostgresVersion   = "12"
+	defaultK8sEnvtestVersion = "1.22.x!"
```

Comment:
Does this always match https://github.com/sapcc/go-makefile-maker/blob/main/internal/renovate/renovate.go#L75 ?